ila/lib/views/addCourseView.dart

```diff
@@ -0,0 +1,212 @@
+import 'package:flutter/material.dart';
+import 'package:flutter/services.dart';
+import 'package:ila/config.dart';
+import 'package:ila/main.dart';
+import 'package:ila_swagger/api.dart';
+
+class AddCourseView extends StatefulWidget {
+  Course course;
+
+  AddCourseView({this.course});
+
+
+  @override
+  _AddCourseViewState createState() => _AddCourseViewState();
+}
+
+class _AddCourseViewState extends State<AddCourseView> {
+  int _id;
```

Comment:
Since an integer has limited space,  the size of the number entered in the course id has to be limited accordingly.
Otherwise the following exception is caused: 
``FormatException: Positive input exceeds the limit of integer``